tests/test_constraint_verifier.py

```diff
@@ -268,3 +275,37 @@ class Solution:
 
     verifier_suite(verifier, same_value, is_value_one,
                    solution, e1, e2, e3, v1, v2, v3)
+
+
+ignored_java_functions = {
+    'equals',
+    'getClass',
+    'hashCode',
+    'notify',
+    'notifyAll',
+    'toString',
+    'wait',
+    'with_constraint_stream_impl_type'
+}
+
+
+def test_has_all_methods():
+    for python_type, java_type in ((ConstraintVerifier, JavaConstraintVerifier),
+                                   (SingleConstraintAssertion, JavaSingleConstraintAssertion),
+                                   (SingleConstraintVerification, JavaSingleConstraintVerification),
+                                   (MultiConstraintAssertion, JavaMultiConstraintAssertion),
+                                   (MultiConstraintVerification, JavaMultiConstraintVerification)):
```

Comment:
Still doesn't catch anything. It should be catching all the new penalizesMore/Less... methods.

Reply:
Those methods are on `MultiConstraintAssertion` and `SingleConstraintAssertion`, so it should be those classes.

Reply:
What do you mean? I am checking for those classes as well, am I not?

Reply:
`ConstraintVerifier` only have `verifyThat` in both Java and Python. So you need `(MultiConstraintAssertion, JavaMultiConstraintAssertion)` and `(SingleConstraintAssertion, JavaSingleConstraintAssertion)` in that list.

Reply:
![image](https://github.com/TimefoldAI/timefold-solver-python/assets/247403/fc9de9dc-60c2-4fcd-9236-1aa3f91be8a1)


And they're not there? I feel like I'm looking at them.

Reply:
Did you do `mvn clean install` on `timefold-solver-core` so the updated classes are used?

Reply:
I did.

Reply:
So, @Christopher-Chianelli, how do I make it do what it's supposed to?
The code is seemingly identical to the one in CS, where it works.

Reply:
I'll investigate.